app/assets/stylesheets/components/table/_table-body.scss

```diff
@@ -0,0 +1,137 @@
+//----------------------------------------
+// classes
+//----------------------------------------
+.table-body {
+  margin-top: rem-calc(30);
+  margin-bottom: rem-calc(24);
+
+  display: block;
+
+  @include breakpoint($medium) { margin-top: 0; }
+
+  &__row {
+    @include table-grid-basic;
+    border: solid 1px $grey-light;
+    margin-top: rem-calc(16);
+    margin-bottom: rem-calc(16);
+    padding: rem-calc(6 0);
+
+    @include breakpoint($medium) { 
+      padding: 0;
+    }
+
+    &:nth-child(even) { 
+      @include breakpoint($medium) { background-color: $white; }
+    }
+  }
+
+  &__cell {
+    margin: 0;
+    padding: rem-calc(18 14);
+    width: 100%;
+
+    @include breakpoint($medium) {
+      padding: rem-calc(16 18);
+      
+      display: inline-grid;
+      align-items: center;
+      justify-content: flex-start;
+    }
+
+    &:first-child { border-left: none; }
+
+    &:before {
+      font-weight: $bold; 
+      margin-right: rem-calc(6);
+
+      @include breakpoint($medium){ display: none; }
+    }
+
+    &:nth-child(n+2) {
+      .table-body--basic & {
+        font-weight: $bold;
+      }
+    }
+    // Add copy for the table header
+    // These are shown on mobile and tablet
+    // Check pame_evalutation.rb for the copy
+    &:nth-child(1) { &:before { content: 'Name:'; } }
+    &:nth-child(2) { &:before { content: 'Committed :'; } }
+    &:nth-child(3) { &:before { content: 'Duration:'; } }
+    &:nth-child(4) { &:before { content: 'Stage:'; } }
+    &:nth-child(5) { &:before { content: ''; } }
+
+    &--actions {
+      padding-right: rem-calc(50);
+
+      display: flex;
+      justify-content: space-between;
+    }
+
+    &--live-state {
+      color: $green;
+    }
+
+    &--draft-state {
+      max-width: rem-calc(280);
+    }
+  }
+
+  &__cell-title { 
+    font-weight: $bold;
+    
+    display: block;
+  }
+
+  &__cell-state {
+    font-weight: $bold;
+  }
+
+  &__cell-draft-info {
+    font-size: rem-calc(10);
+    line-height: rem-calc(10);
+
+    display: block;
+  }
+
+  &__cell-action {
+    &--destroy {
+      cursor: pointer;
+    }
+  }
+
+  &__cell-action-icon {
+    height: rem-calc(30);
```

Comment:
The heights are a bit off because of the whitespace in the image files - should probably trim them (or just resize and do it by eye)